test/.env

```diff
@@ -0,0 +1,21 @@
+INDY_NETWORK_NAME=idunion_local_test
+
+INDY_NODE_IP_1=10.133.133.1
+INDY_NODE_PORT_1=9701
+INDY_CLIENT_IP_1=10.133.133.1
+INDY_CLIENT_PORT_1=9702
+
+INDY_NODE_IP_2=10.133.133.2
+INDY_NODE_PORT_2=9703
+INDY_CLIENT_IP_2=10.133.133.2
+INDY_CLIENT_PORT_2=9704
+
+INDY_NODE_IP_3=10.133.133.3
+INDY_NODE_PORT_3=9705
+INDY_CLIENT_IP_3=10.133.133.3
+INDY_CLIENT_PORT_3=9706
+
+INDY_NODE_IP_4=10.133.133.4
+INDY_NODE_PORT_4=9707
+INDY_CLIENT_IP_4=10.133.133.4
+INDY_CLIENT_PORT_4=9708
```

Comment:
you only need to set the IP Adress if you want to prevent access from other adresses. for a local testnet this may or may not be reasonable, but setting it to 0.0.0.0 will illiminate one potential source of errors

Reply:
Thanks, I changed that and fixed the wrong port assignments in docker-compose.yaml